doc/source/index.rst

```diff
@@ -29,14 +29,45 @@ Appliance ?
 -----------
 
 An appliance is a ready to use image of an operating system including
-a preconfigured application for a specific use case. The appliance is
+a pre-configured application for a specific use case. The appliance is
 provided as an image file and needs to be deployed to, or activated in
 the target system or service.
 
-In KIWI the appliance is specified by a collection of human readable files
-in a directory, also called as the `image description`. At least one XML
-file :file:`config.xml` or :file:`.kiwi` is required. In addition
-there may be as well other files like scripts or configuration data.
+KIWI can create appliances in various forms: beside classical installation
+ISOs and images for virtual machines it can also build images that boot via
+PXE or vagrant boxes.
+
+In KIWI the appliance is specified via a collection of human readable files
+in a directory, also called the `image description`. At least one XML file
+:file:`config.xml` or :file:`.kiwi` is required. In addition there may as
+well be other files like scripts or configuration data.
+
+
+Use cases
+---------
+
+Not convinced yet? You can find a selection of the possible uses cases
+below:
+
+* You are a system administrator and wish to create a customized installer
+  for your network that includes additional software and your organizations
+  certificates? KIWI allows you to select which packages will be included
+  in the final image. On top of that you can add files to arbitrary
+  locations in the filesystem, e.g. to include ssl or ssh keys. You can
+  also tell KIWI to create an image that can be booted via PXE, so that you
+  don't even have to leave your desk to reinstall a system.
+
+* You want to create a custom spin of your favorite Linux distribution with
+  additional repositories and packages that are not present by default?
+  With KIWI you can configure the repositories of your final image via the
+  image description and tweak the list of packages that are going to be
+  installed to match your target audience.
+
+* The Raspberry Pi that is coordinating your home's IoT devices got very
```

Comment:
An unknown acronym should be explained at least once: ... your home's Internet of Thing (IoT) devices...